articles/azure-app-configuration/howto-best-practices.md

```diff
@@ -88,8 +88,10 @@ App Configuration is regional service. For applications with different configura
 
 ## Client Applications in App Configuration 
 
-Excessive requests to App Configuration can result in throttling or overage charges. Applications take advantage of the caching and intelligent refreshing currently available to optimize the number of requests they send. This process can be mirrored in high volume client applications by avoiding direct connections to the configuration store. Instead, client applications connect to a custom service, and this service communicates with the configuration store. This proxy solution can ensure the client applications do not approach the throttling limit on the configuration store. For more information on throttling, see [the FAQ](./faq.yml#are-there-any-limits-on-the-number-of-requests-made-to-app-configuration).  
+When it comes to using App Configuration in client applications, there are two major factors to consider. Firstly, if you are using the connection string in a client application, you may risk exposing the access key of your App Configuration store to the public. Secondly, due to the typical scale of client application instances, it may cause excessive requests to your App Configuration store that can result in overage charges or throttling (for more information on throttling, see [the FAQ](./faq.yml#are-there-any-limits-on-the-number-of-requests-made-to-app-configuration)).
+
+You are recommended to use a proxy service between your client applications and your App Configuration store. The proxy service can securely authenticate with your App Configuration store without worrying about leaking its authentication information. It can be built using one of the App Configuration provider libraries to take advantage of the caching and refreshing capabilities to optimize the number of requests it sends (for more information, see topics in Quickstarts and Tutorials sections). This proxy service can then serve the configuration from its cache to your client applications. This approach avoids the two potential issues discussed previously.
```

Comment:
Could we also mention that this can be set up to work with multiple configuration stores? Customers usually have separate dev and prod config stores and would want to know that they can configure the proxy solution to work with this. 
And can we link to the relevant quickstart (/azure/azure-app-configuration/quickstart-aspnet-core-app) and push refresh (/azure/azure-app-configuration/enable-dynamic-configuration-dotnet-core-push-refresh) here? that gives people a specific next step rather than having to find the tutorial/quickstarts themselves.

Reply:
Using different stores for dev/prod is not a specific topic for client apps. Maybe you can share with me more details about the customer questions and we can see what's the best to do.

I know this is not ideal, but the proxy service can be a java app, a full framework app, or even a Function app if someone wants to. It's up to users whether they want the proxy service to load config dynamically. It's not practical to link almost all documents under Quickstarts and Tutorial here.

Reply:
Yes - it does not have to be dev/prod specifically, but mentioning that this can be configured to work with multiple config stores if needed would be helpful. 

Yes, we should then not link to any specific quickstarts, it should be flexible. I think it would be a good idea though to create a "golden path" for our tutorials where someone who is building a proxy service should probably be using dynamic config or a push model etc. It doesn't have to be the only tutorial listed, what is there now works, I just don't want users then go to the tutorial section and get overwhelmed of what to implement next.

Reply:
I'm not sure how to mention dev/prod without distracting the key topic here. Any suggestions? Honestly speaking, I think dev/prod is a different topic. Please help me to understand why it should be mentioned here. We have an FAQ for it. If there are customer questions, we should point customers to that.
https://docs.microsoft.com/en-us/azure/azure-app-configuration/faq#how-should-i-store-configurations-for-multiple-environments--test--staging--production--and-so-on-

Reply:
BTW, I do agree with the idea of having "golden path" tutorials. They cover one or two typical E2E scenarios. I think you also have the idea to show people how they can estimate the number of requests in a typical setup. Those will be very helpful.

Reply:
Yes, I do not think that dev/prod specifically needs to be mentioned here, it is too specific. For now, the FAQ will be good.